README.md

```diff
@@ -416,7 +416,54 @@ begin
 
   readln();
 end.
-      
+```
+
+</td></tr>
+</table>
+
+### ruapu with C++
+
+<table>
+
+<tr><td>
+
+Compile ruapu library
+
+```shell
+cd cpp
+g++ test_ruapu.cpp -o ruapu
+```
+
+</td>
+<td>
+
+Use ruapu in C++
+
+```c++
+#include <iostream>
+#include <vector>
+
+#include "ruapu.hpp"
+
+int main()
+{
+    ruapu my_ruapu;
```

Comment:
Refactor all relevant lines to use class methods (as well as the lines in the actual test file, `test_ruapu.cpp`)